docs/resources/target_group.md

```diff
@@ -0,0 +1,75 @@
+---
+subcategory: "Application Load Balancer"
+layout: "ionoscloud"
+page_title: "IonosCloud: target_group"
+sidebar_current: "docs-resource-target_group"
+description: |-
+Creates and manages IonosCloud Target Group.
```

Comment:
I think this is invalid yaml and won't show on gitbook correctly

Reply:
Yeah, missed this one :))